playerUi/timelineMarkersLive.html

```diff
@@ -0,0 +1,178 @@
+    }
+
+    .content {
+      margin-bottom: 10em;
+    }
+
+    h1,
+    h2,
+    h3,
+    p {
+      font-weight: 300;
+      text-align: center;
+      margin: 40px;
+    }
+
+    #player {
+      max-width: 900px;
+      width: 90%;
+      margin: auto;
+      -webkit-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+    }
+
+    a {
+      color: #97d9ef;
+      font-weight: 400;
+      text-decoration: none;
+    }
+
+    a:hover {
+      color: #fff;
+    }
+
+    @media (max-width: 800px) {
+      .logo {
+        width: 100%;
+      }
+
+      .title {
+        display: none;
+      }
+    }
+  </style>
+</head>
+
+<body>
+  <div id="wrapper">
+    <div id="banner">
+      <div class="logo"><img src="../images/bitmovin-logo.png"></div>
+      <div class="title">
+        <h1>Timeline markers</h1>
+      </div>
+      <div class="clear"></div>
+    </div>
+    <div class="container">
+      <h1>HTML5 Adaptive Streaming Player for MPEG-DASH & HLS</h1>
+      <h2>Your videos play everywhere with low startup delay, no buffering and in highest quality.</h2>
+      <div class="content">
+        <div class="player-wrapper">
+          <div id="player"></div>
+        </div>
+        <div class="description">
+          <p>For more information about the bitmovin player, please have a look at our online <a
+              href="//bitmovin.com/docs" target="_blank">Documentation</a>.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    var conf = {
+      key: "YOUR-PLAYER-LICENSE-KEY",
+      ui: false,
+      playback: {
+        autoplay: true,
+        muted: true,
+      },
+    };
+  
+    var sourceConfig = {
+      title: "Bitmovin Player",
+      description: "This demo showcases the possibility to define markers in the timeline for live streams",
+      'dash': 'https://livesim.dashif.org/livesim/testpic_2s/Manifest.mpd',
+    };
+
+    var player = new bitmovin.player.Player(document.getElementById('player'), conf);
+    var myUiManager = new bitmovin.playerui.UIFactory.buildDefaultUI(player);
+
+    player.load(sourceConfig).then(function () {
+      console.log('Successfully loaded source');
+    }, function () {
+      console.log('Error while loading source');
+    });
+
+    player.on('ready', () => {
+      // Timeline markers for live stream uses timestamps 
+      const currentTime = player.getCurrentTime();
+      const imageUrl = './../images/bit-fav.png';
+  
+      const markers = [
+        { time: currentTime - 40, duration: 10, title: 'Marker 1', imageUrl },
+        { time: currentTime - 20, duration: 5, title: 'Marker 2' },
+        { time: currentTime + 5, duration: 10, title: 'I come from the future!' },
+      ];
+
+      markers.forEach(marker => {
+        myUiManager.addTimelineMarker(marker);
+      });
+    });
+  </script>
+</body>
+
+</html>
```

Comment:
Please add an empty line after this.

Reply:
Addressed in https://github.com/bitmovin/bitmovin-player-web-samples/pull/46/commits/513e7f029531bb7d6aafe801d0bf7513cadb4549